From b28a22fe227c6b04825d1b1969f23ffb77ed5e57 Mon Sep 17 00:00:00 2001
From: razorloves <razorloves@gmail.com>
Date: Tue, 12 Mar 2019 00:09:46 -0500
Subject: [PATCH 24/26] overlay: Move managed profiles overlay to
 frameworks/base

The ManagedProvisioning xml's were moved in Pie.
This fixes 'Android for Work'.

REF: https://github.com/LineageOS/android_packages_apps_ManagedProvisioning/commit/ebe73b6e94e65fc82fde2c482844a05deb84e806
     https://github.com/LineageOS/android_frameworks_base/commit/b6a9f94d6718894fdd382708a9d87696f055e71e

Change-Id: If911b4ee1890600e47af73c7fe0fcca4dc73eb26
---
 .../res}/res/values/vendor_required_apps_managed_profile.xml  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
 rename overlay/common/{packages/apps/ManagedProvisioning => frameworks/base/core/res}/res/values/vendor_required_apps_managed_profile.xml (87%)
 mode change 100755 => 100644

diff --git a/overlay/common/packages/apps/ManagedProvisioning/res/values/vendor_required_apps_managed_profile.xml b/overlay/common/frameworks/base/core/res/res/values/vendor_required_apps_managed_profile.xml
old mode 100755
new mode 100644
similarity index 87%
rename from overlay/common/packages/apps/ManagedProvisioning/res/values/vendor_required_apps_managed_profile.xml
rename to overlay/common/frameworks/base/core/res/res/values/vendor_required_apps_managed_profile.xml
index 212b5c95..f276b402
--- a/overlay/common/packages/apps/ManagedProvisioning/res/values/vendor_required_apps_managed_profile.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/vendor_required_apps_managed_profile.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
 /**
- * Copyright (C) 2014 The Android Open Source Project
+ * Copyright (C) 2017 The Android Open Source Project
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -19,7 +19,7 @@
 <resources>
     <!-- A list of apps to be retained in the managed profile by a particular vendor.
             Takes precedence over the disallowed apps lists. -->
-    <string-array name="vendor_required_apps_managed_profile">
+    <string-array translatable="false" name="vendor_required_apps_managed_profile">
         <item>com.android.vending</item> <!-- Google Play -->
         <item>com.google.android.gms</item> <!-- Required by Play -->
         <item>com.android.contacts</item> <!-- AOSP Contacts -->
-- 
2.17.1

